client/landing/gutenboarding/onboarding-block/plans/index.tsx

```diff
@@ -20,6 +19,7 @@ import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
 import { PLANS_STORE } from '../../stores/plans';
 import { Step, usePath } from '../../path';
 import { useFreeDomainSuggestion } from '../../hooks/use-free-domain-suggestion';
+import AsyncLoad from '../../../../components/async-load';
```

Comment:
We have these aliased

```suggestion
import AsyncLoad from 'components/async-load';
```